include/BareCpper/ATsamd/TwiAsync_ATsamd5x.hpp

```diff
@@ -641,10 +642,10 @@ namespace BareCpper
         Service service_; ///< Service state
     };
 
-    TwiAsync* TwiAsync::sercom_s[SERCOM_INST_NUM] = {};
+    inline TwiAsync* TwiAsync::sercom_s[SERCOM_INST_NUM] = {};
 } //END: BareCpper
 
-
+// multiple inclusions problem
```

Comment:
Further This also is explicit only Sercom2! It feels like the user needs to instantiate these or possibly using the alias attribute could work i.e. `samd51-csp-src\samd51a\gcc\gcc\startup_samd51.c` uses this to setup the Vector table to `Dummy_Handler` by default and it may be viable to replace/specialise this behaviour to call directly to our C++ static method acircumventing the need for the c-global function (TBD)

Reply:
Yes, changing from header-only library will require quite a few changes I think. I've moved the handlers in a separate header file that should be included only in one source file, that also has included the `Twi` header from `BareCpper`. Not ideal, definitely not for merging.